ratis-docs/src/site/markdown/index.md

```diff
@@ -17,12 +17,43 @@
 
 # Apache Ratis
 Apache Ratis is a highly customizable Raft protocol implementation in Java.
-Raft is a easily understandable consensus algorithm to manage replicated state.
-Apache Ratis could be used in any Java application where state should be replicated between multiple instances.
+[Raft](https://raft.github.io/) is an easily understandable consensus algorithm to manage replicated state. 
+
+The Ratis project was started at 2016,
+entered Apache incubation in 2017,
+and graduated as a top level Apache project on Feb 17, 2021.
+Originally, Ratis was built for using Raft in [Apache Ozone](https://ozone.apache.org)
+in order to replicate raw data and to provide high availability.
+The correctness and the performance of Ratis has been heavily tested with Ozone.
+
+Unlike many other raft implementations, 
+Ratis is designed to be pluggable, 
+it could be used in any Java application where state should be replicated between multiple instances.
```

Comment:
Combine two pluggable related paragraphs.
```
-it could be used in any Java application where state should be replicated between multiple instances.
-
-## Ratis Features
-
-Since Ratis is designed to be pluggable in any Java applications,
+it could be used in any Java applications
+where state should be replicated between multiple instances.
```
After all the changes, the section will look like below
```
## Pluggability

Unlike many other raft implementations,
Ratis is designed to be pluggable,
it could be used in any Java applications
where state should be replicated between multiple instances.
Ratis provides abstractions over Raft protocol for users,
which make Raft library fully decoupled from the applications.
```